src/main/java/org/example/homework6/Calculator.java

```diff
@@ -0,0 +1,77 @@
+package org.example.homework6;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class Calculator {
+    String decisionValue;
+    Scanner scanner = new Scanner(System.in);
+
+    public void calculator() {
```

Comment:
```suggestion
    public void calculate() {
```
method name should describe an action